src/components/ChangePlan/PlanCalculator.js

```diff
@@ -186,74 +215,120 @@ const PlanCalculator = ({
-                  {planData.discount?.discountPercentage
-                    ? Math.round(
-                        getPlanFee(planData.plan) *
-                          (1 - planData.discount?.discountPercentage / 100) *
-                          planData.discount.monthsAmmount,
-                      )
-                    : getPlanFee(planData.plan)}
-                </span>
-              </section>
-            </div>
-            <div style={{ marginTop: '40px' }}>
-              <span className="col-lg-1">
-                <Link to={`/plan-selection${safePromoId ? `?promo-code=${safePromoId}` : ''}`}>
-                  {' '}
-                  &lt; &lt; Volver a Planes
-                </Link>
-              </span>
-              <span className="col-lg-1">
-                <a
-                  className="dp-button button-medium primary-green"
-                  href={
-                    _('common.control_panel_section_url') +
-                    `/AccountPreferences/UpgradeAccountStep2?IdUserTypePlan=${planData.plan.id}&fromStep1=True` +
-                    `${planData.discount?.id ? `&IdDiscountPlan=${planData.discount?.id}` : ''}` +
-                    `${safePromoId ? `&PromoCode=${safePromoId}` : ''}`
-                  }
-                >
-                  Contratar
-                </a>
-              </span>
+            <section className="tab--container col-sm-12">
+              <article className="tab--content active">
+                <div className="dp-container">
+                  <div className="dp-rowflex">
+                    <div className="dp-calc-box">
+                      <div className="col-md-6 col-sm-12">
+                        <article class="dp-box-shadow">
+                          <Slider
+                            planDescriptions={state.descriptions}
+                            defaultValue={0}
+                            handleChange={(index) => {
+                              dispatchPlanData({
+                                type: actionTypes.UPDATE_SELECTED_PLAN,
+                                indexPlan: index,
+                              });
+                            }}
+                          />
+                          <hr />
+                          {state.discountsList ? (
+                            <Discounts
+                              discountsList={state.discountsList}
+                              handleChange={(discount) => {
+                                dispatchPlanData({
+                                  type: actionTypes.UPDATE_SELECTED_DISCOUNT,
+                                  idDiscount: discount.id,
+                                });
+                              }}
+                            />
+                          ) : (
+                            <></>
+                          )}
+                        </article>
+                      </div>
+                      <div className="col-md-6 col-sm-12">
```

Comment:
Perhaps all price handling could become something like a `price` Component or `priceWithDiscount` component, to give more meaning to this section. You could have something like this:
`
<SelectedPlanPrice>
   <RegularPlanPrice/>
   <PlanPriceWithPromotions/>
</SelectedPlanPrice>
`
Does it make sense?

Reply:
Yes, I can try to improve it, but I don't know if this is something to resolve in this PR. Maybe create a ticket to isolate price section in a new element, what do you think?

Reply:
This will be resolve in [DW-529](https://makingsense.atlassian.net/jira/software/projects/DW/boards/16?selectedIssue=DW-529)